pkg/users/users.go

```diff
@@ -35,10 +35,7 @@ func GetUsers(jxClient versioned.Interface, ns string) (map[string]*jenkinsv1.Us
 
 // CreateUser creates a new default User
 func CreateUser(ns string, login string, name string, email string) *jenkinsv1.User {
-	id := login
-	if email != "" {
-		id = kube.EmailToK8sID(email)
-	}
+	id := kube.ToValidName(login)
```

Comment:
Is `login` being empty here _ever_ a possibility?

Reply:
*everything* is a possibility and it is a global function.  we will find out if anyone does pass a `nil` login soon enough - but this was how the old code worked.